bigquery_etl/parse_udf.py

```diff
@@ -13,7 +13,10 @@
 import sqlparse
 
 
-UDF_RE = re.compile(r"udf_[a-zA-z0-9_]+")
+UDF_DIRS = ("udf", "udf_js")
+UDF_CHAR = "[a-zA-z0-9_]"
+UDF_RE = re.compile(f"udf_{UDF_CHAR}+")
+PRESISTENT_UDF_RE = re.compile(f"(udf{UDF_CHAR}*).({UDF_CHAR}+)")
```

Comment:
```suggestion
PRESISTENT_UDF_RE = re.compile(f"(udf{UDF_CHAR}*)\.({UDF_CHAR}+)")
```

Reply:
good catch, this needs to use a raw string to preserve the `\` as well
```suggestion
PRESISTENT_UDF_RE = re.compile(fr"(udf{UDF_CHAR}*)\.({UDF_CHAR}+)")
```